decidim-core/app/controllers/concerns/decidim/locale_switcher.rb

```diff
@@ -55,6 +56,23 @@ def available_locales
       def default_locale
         @default_locale ||= (current_organization || Decidim).public_send(:default_locale)
       end
+
+      # Save current locale in session variable
+      # to prevent unnecessary redirects
+      def set_locale
+        logger.debug "* Actual locale: #{I18n.locale}"
+        logger.debug "* Accept-Language: #{request.env["HTTP_ACCEPT_LANGUAGE"]}"
+        logger.debug "* params[:locale]: #{params[:locale]}"
```

Comment:
Could you please remove the debug lines?

Reply:
Done